soepy/python/pre_processing/model_processing.py

```diff
@@ -54,16 +54,13 @@ def transform_old_init_dict_to_df(init_file_name):
     return model_params_df
 
 
-# Pre-processing of model parameters to be estimated
-
-
 def read_model_params_init(model_params_df):
     """Reads in specification of model parameters
     from a pickled data frame and saves parameters as named tuple."""
 
     # Transform data frame to dictionary
     model_params_dict = {
-        l: model_params_df.xs(l)["value"].to_dict()
+        l: model_params_df.loc[l]["value"].to_dict()
```

Comment:
Please use it like this `l: model_params_df.loc[l, "value"].to_dict()`

Reply:
Done.